modules/store/src/wrappers/fileStorage.ts

```diff
@@ -49,12 +51,13 @@ export class FileStorage implements WrappedStorage {
     return path.join(this.fileDir, fileName);
   }
 
-  async getItem(key: string): Promise<string | undefined> {
+  async getItem(key: string): Promise<any | undefined> {
```

Comment:
Perhaps we could also improve these `any` types with type injection

Reply:
That's what I was attempting but it was becoming difficult. Will take another look.